RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -90,7 +81,6 @@ final class ManageSubscriptionsViewModel: ObservableObject {
             self.actionWrapper = actionWrapper
             self.loadPromotionalOfferUseCase = loadPromotionalOfferUseCase
             ?? LoadPromotionalOfferUseCase(purchasesProvider: purchasesProvider)
-            self.state = .success
```

Comment:
doesn't removing this break the view? maybe this should be done in another PR?

Reply:
The state is not being used anywhere, besides in tests that just check the initial state 😅 
I think it was meant to be used, but ended up in lost in the ether. I can bring it back once the screen refactor is done.